modules/calendar/add_edit_event.php

```diff
@@ -939,8 +939,8 @@ function setEventDate($start_date, $recurrence)
 
     //-------------------------------------
     //(CHEMED)
-    //Set default facility for a new event based on the given 'userid'
-    if ($userid) {
+    //Set default facility for a new event based on logged in user, not provider (event slot)
```

Comment:
Ooops.  Looks like maybe the array element session variable should be authUserId and not authId.

Reply:
Did a console.log test using `<?php echo "<script>console.log('{$_SESSION['authId']}')</script>"; ?>` &
    `<?php echo "<script>console.log('{$_SESSION['authUserId']}')</script>"; ?>`, the latter is blank while former corresponds to an user id.

Reply:
AGreed...I am casting around looking for problems, because we can't get any appointments to display on my branch test or Marcus Bowens'.
Periodically receiving the "error occurred while fetching appointments."  Looking into everything I can right now.

Reply:
@aethelwulffe that error is usually caused by the get_provider_events.php not the add_edit_event.php